telemetry/telemetry.go

```diff
@@ -127,6 +129,7 @@ func reportUsageData(ctx sessionctx.Context, etcdClient *clientv3.Client) (bool,
 	}
 
 	req.Header.Add("Content-Type", "application/json")
+	logutil.BgLogger().Info(fmt.Sprintf("Uploading telemetry data to %s", apiEndpoint))
```

Comment:
How about also adding the report interval in the log?

Reply:
Maybe that should be logged at startup? e.g.:
```telemetry=enabled apiEndpoint="https://telemetry.pingcap.com", interval=6h```? 

And then keep this line as is when actually sending the report?

Reply:
Yep. This way LGTM.

Reply:
@lzmhhh123 I've added the reporting of the config on startup. This also does an initial run on startup now.